Startup.cs

```diff
@@ -34,7 +33,7 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment environment)
+        public void Configure(IApplicationBuilder app, IHostingEnvironment environment)
```

Comment:
The old code is right. This should be reverted.

Reply:
@bradygaster You probably need to use a newer aspnet core runtime

Reply:
Sorry about that - noticed a build error and went the wrong way on it. I'll update my own environment this time. :)